src/Components/NavBar/CategoryItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './NavBar-click.scss';
+
+const CategoryItem = ({CategoryList}) => {
+	const { name, text, click } = CategoryList;
```

Comment:
props도 파라미터로 넘겨받는 변수이기 때문에
다른 코드들과 동일하게 lower camelcase 를 지켜서 작성하는 것을 추천 드릴게요!

```suggestion
const CategoryItem = ({categoryList}) => {
	const { name, text, click } = categoryList;
```